### PR TITLE
feat(json): prints result as JSON if --json flag is passed alongside URL

### PR DESCRIPTION
**Pull request by rajkumaar23**
## Description

Prints output as JSON if --json flag is passed alongside URL
Usage : is-website-vulnerable http://willhaben.at/ --json

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Related Issue

#4 

## Motivation and Context

The default is to print out the results to the console but now add a --json flag to spit out the data in a JSON format.

## Checklist:

- [x] I have updated the documentation (if required).
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] I added a picture of a cute animal cause it's fun

### CODE REVIEW

**Comment by rajkumaar23** [comment]
> @rajkumaar23 I would like to suggest some updates:
> 
>     * can you add the `yargs` dependency and use it to support the `--json` flag?
> 
>     * instead of changing `RenderConsole` can you add another class called `RenderJson` and in the CLI code (`./bin/is-website-vulnerable.js`) use either one of them depending on whether the --json flag was provided or not.
> 
>     * we don't need `package-lock.json` in this PR because we use yarn, can you remove it (and if you also want you can add it to .gitignore so it doesn't get added by others)

Done @lirantal.

**Comment by rajkumaar23** [comment]
Hey, is that fine ? @lirantal

**Comment by lirantal** [comment]
@rajkumaar23 amazing job ✨❤️
do you know how to remove `package-lock.json` from the PR though? since that is still part of the commits you have. Also you need to commit the `yarn.lock` changes. Let me know if you need help with that.

Other than this - good to go and I'll merge when you update me! Great job again 👏

**Comment by rajkumaar23** [comment]
I don't understand the checks that failed now. Could you help ? @lirantal

File: src/RenderJson.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/* eslint-disable security/detect-object-injection */
+/* eslint-disable indent */
+'use strict'
+
+module.exports = class RenderConsole {
```
Comment (lirantal) [inline src/RenderJson.js]: @rajkumaar23 I noticed this one now, can you rename it to `RenderJson` ?

**Comment by lirantal** [comment]
@rajkumaar23 don't sweat about the code coverage issues, we'll figure it out in future PRs. I just now noticed that the class name for RenderJson is still called RenderConsole. Can you just commit a fix for that quickly and I'll merge away?

**Comment by lirantal** [comment]
Looks great, thanks!

**Pull request merged**